src/tlo/methods/healthsystem.py

```diff
@@ -904,6 +913,12 @@ def initialise_simulation(self, sim):
         sim.schedule_event(HealthSystemChangeMode(self),
                            Date(self.parameters["year_mode_switch"], 1, 1))
 
+        # Schedule recurring event which will rescale daily capabilities at regular intervals.
+        # Note: if want to use Demography's popsize_by_year for current year too (see options in DynamicRescalingHRCapabilities),
+        # need to ensure that the DynamicRescalingHRCapabilities event takes place *after* the DemographyLoggerEvent has been called.
+        # Could achieve this for example by scheduling the former to happen on 2nd of Feb..
```

Comment:
We should discuss how we want to combine the pop growth with the dynamic_HR_scaling_factor, because to me considering e.g. a combined fixed 5% growth on top of pop growth would be ratio_pop_growth x 1.05 (as I had originally coded), not ratio_pop_growth + 1.05, but maybe I'm missing something

Reply:
my mistake. Multiplication is right.